crt_portal/api/filters.py

```diff
@@ -0,0 +1,103 @@
+# Class to handle filtering data by supplied query params, providing the params are valid.
+
+from actstream.models import Action
+import urllib.parse
+from datetime import datetime
+
+from utils.datetime_fns import _change_datetime_to_end_of_day
+
+# To add a new filter option, add the field name and expected filter behavior
+filter_options = {
+    'start_date': '__gte',
+    'end_date': '__lte'
+}
+
+
+def contacts_filter(querydict):
+    kwargs = {}
+    filters = {}
+    action_qs = Action.objects.filter().all()
+    contact_qs = Action.objects.filter(verb='Contacted complainant:').all()
+    total_actions = len(action_qs)
+    total_contacts = len(contact_qs)
+    total_emails_counter = {'CRM - R1 Form Letter': 0,
```

Comment:
Billy and I discussed this while on a call.  Instead of hardcoding the email titles in `total_emails_counter`, I will instead pull the email titles from `/api/responses`.